lib/entities/Article.ts

```diff
@@ -23,6 +23,7 @@ export interface BasicArticle {
     tags?: BasicArticleTag[];
     imageUrl: string;
     importanceScore: number;
+    isLocalScore?: number;
```

Comment:
Let's change to "localScore" to correspond to float ("is" suggests boolean type)